packages/ui-components/src/components/getting-started/getting-started-step-overview.ts

```diff
@@ -0,0 +1,94 @@
+import { BaseElement } from '../../base';
+import { TemplateResult, css, html } from 'lit';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+import TrailOne from '../../assets/trail-1.svg?raw';
+import TrailTwo from '../../assets/trail-2.svg?raw';
+import TrailThree from '../../assets/trail-3.svg?raw';
+
+export class GettingStartedStepOverview extends BaseElement {
+  static styles = [
+    ...BaseElement.styles,
+    css`
+      .grid-template {
+        grid-template-rows: 250px 250px 390px;
+        grid-template-columns: 1fr 1fr 200px 1fr 1fr;
+      }
+
+      .trail-one {
+        margin: 0 0 -140px 20px;
+      }
+
+      .trail-two {
+        margin: 0 20px -15px 0;
+      }
+
+      .trail-three {
+        margin: 0 0 -110px -80px;
+      }
+    `,
+  ];
+  steps: { title: string; description: TemplateResult }[] = [
+    {
+      title: '🔒 Sign in through Github',
+      description: html`
+        You should see a list of all your <u>public</u> repositories.
+      `,
+    },
+    {
+      title: '✅ Enable your repository',
+      description: html`
+        Click <u>enable</u> next to the repository you want to use with
+        <i>Stryker Dashboard</i>.<br /><br />
```

Comment:
Do not use `<br>`.

Reply:
Why not? 

Using two spans and adding margins in between is not the way to go here since it is just a textblock with a newline. Did you mean something else than I am thinking of?

Reply:
I would suggest using multiple ‘<p>’ tags. This way it is more semantic and it won’t give me the shivers (brrrrr).